easybuild/easyconfigs/m/MUSCLE/MUSCLE-5.1.0-add_Aarch64_support.patch

```diff
@@ -0,0 +1,15 @@
+Add support for aarch64
+See https://github.com/EESSI/software-layer/pull/1032#issuecomment-2813036581
+author: Lara
```

Comment:
@laraPPr I'm pretty sure there's more than 1 Lara in the world, so let's use full name here (+ affiliation, since Peeters isn't exactly uncommon either ;) )

```suggestion
author: Lara Peeters (UGent)
```

Note that this implies also changing the checksum for the patch file in the easyconfig